dbclient/test/ClustersClientTest.py

```diff
@@ -0,0 +1,63 @@
+import json
+import os
+import unittest
+from unittest.mock import MagicMock
+from dbclient import ClustersClient
+from dbclient.test.TestUtils import TEST_CONFIG
+
+
+class TestClustersClient(unittest.TestCase):
+
+    def test_log_cluster_ids_and_original_creators(self):
+        TEST_OUTPUT_CREATOR_USER_IDS_FILE = "test_output_original_creator_user_ids.log"
+        TEST_OUTPUT_CLUSTER_IDS_FILE = "test_output_cluster_ids_to_change_creator.log"
+        EXPECTED_CREATOR_USER_IDS_FILE = "expected_original_creator_user_ids.log"
+        EXPECTED_CLUSTER_IDS_FILE = "expected_cluster_ids_to_change_creator.log"
+
+        TEST_DIR = "dbclient/test/"
+
+        clustersClient = ClustersClient(TEST_CONFIG)
+        clustersClient.get_export_dir = MagicMock(return_value =TEST_DIR)
+        clustersClient.get_cluster_list = MagicMock(return_value=[
+            {
+                'cluster_name': "cluster_1",
+                'cluster_id': 'new_cluster_id_1',
+                'creator_user_name': 'new_cluster_creator_user_name_1'
+            },
+            {
+                'cluster_name': "cluster_2",
+                'cluster_id': 'new_cluster_id_2',
+                'creator_user_name': 'new_cluster_creator_user_name_2'
+            },
+            {
+                'cluster_name': "cluster_3",
```

Comment:
Also coverage for the current creator is correctly set?

Reply:
yeah, now that I changed the logic a little bit, it will always output even if the creator doesn't change. 
(because barely will there be any case where the new creator is the same as the original creator)

Reply:
But let me still add the case

Reply:
done